### DISCUSSION TITLE
How to get the model browser back

### DISCUSSION THREAD

**Discussion by j9440044** [vladmandic/sdnext#2881] [Q&A]
How do you get the model browser back? This thing
![image](https://github.com/vladmandic/automatic/assets/74937260/16a6d2b2-5fc2-4a5b-ae18-a66c21cb2733)

**Comment by kupua775** [comment]
Click the networks button just above your circle there. It will expand out the panel and inside will be tabs to swap between models, lora, embeddings etc. The burger stack will change view modes. You can update thumbnails from civitai within the panel as well.
